src/components/backend-ai-app-launcher.ts

```diff
@@ -36,6 +37,7 @@ export default class BackendAiAppLauncher extends BackendAIPage {
   @property({type: Object}) jobs = Object();
   @property({type: Object}) controls = Object();
   @property({type: Array}) appSupportList;
+  @property({type: Array}) preOpenPortList;
```

Comment:
minor request: Could we change the name of property `preOpenPortList` to `preOpenedPortList` for better understanding?